modules/chess

```diff
@@ -1 +1 @@
-Subproject commit 60ace8e3bfc63b50ff250a65f70f40c7109b7872
+Subproject commit b833ddd619109ce748b2fa35e3f9b3f2018c8b10
```

Comment:
This hash will become outdated on merge of ornicar/scalachess#49. @ornicar has to update this again himself when the other PR is merged by a slave commit.